openreview/conference/invitation.py

```diff
@@ -335,6 +336,37 @@ def __init__(self, conference, reveal_authors, reveal_submission, hide_fields=[]
             }
         )
 
+
+class PaperWithdrawSuperInvitation(openreview.Invitation):
+
+    def __init__(self, conference):
+        content = invitations.withdraw.copy()
+        exp_date = tools.datetime_millis(conference.submission_stage.withdraw_submission_exp_date) if conference.submission_stage.withdraw_submission_exp_date else None
+        super(PaperWithdrawSuperInvitation, self).__init__(
+            id=conference.get_invitation_id("Paper_Withdraw"),
```

Comment:
Could you call it `Withdraw` only so it matches the same name with the PaperX/-/Withdraw invitations?